### DISCUSSION TITLE
Philosophical Musings Concerning 2/21's WG Meeting

### DISCUSSION THREAD

**Discussion by gregfowlerphd** [JisooSeo/101-WG-Discussions#11] [General]
The discussion in this week's Ontology WG meeting, which Carter led, sparked some philosophical thoughts in me that I didn't get a chance to express during the meeting itself, so I thought I'd post about them here.

If you'll recall, Carter went through a series of examples and asked how we would model each example in BFO. One example was 141 Park Hall. Now my initial inclination was that this room should be modeled as a site. However, Michael Rabenberg suggested that it shouldn't be, since sites are immaterial entities and you can damage the room by damaging a material entity--namely, the wall. Instead, he suggested that the room should be modeled as a material entity.

While I think that Michael is clearly right that damaging the wall damages the room, I don't think this shows that the room isn't a site. Perhaps that's just how you damage a site: i.e., by damaging the material entity that is its host. After all, the way you change the shape of, say, a hole (which is an immaterial entity), is by changing the shape of its host (which is a material entity).

(I'm reminded here of an insightful discussion in the introduction to Richard Cartwright's Philosophical Essays concerning whether propositions can be written on a blackboard, if anyone's familiar with it.*)

A second example to which Carter drew our attention was that of a boring seminar. One or two WG members suggested that for something to be boring, there have to be agents involved who are bored by it. I don't think this is right (although I also suspect I won't convince anyone). In my view, paint drying is just boring, whether or not anyone's around to watch it happening. Boringness is, in my view, a feature something has in virtue of which it is *appropriate* or *fitting* to be bored by it--in other words, a feature that makes something boredom-worthy. As such, I think something can be boring even if there are no agents involved. Moreover, I think that something, a seminar for example, can be boring even if none of the agents involved are bored by it--indeed, even if they're positively fascinated by it. They might simply be having an inappropriate response.

(Perhaps it's worth noting that my view about boringness doesn't seem too far from the commonsensical view. My guess is that plenty of undergrads who hated their philosophy classes would agree that a graduate-level philosophy seminar can be boring even if none of its participants are bored by it.)

[*I'd be surprised if anyone else were, but then about half my dissertation (which is available on my PhilPeople page if anyone's interested) is about Cartwright--particularly about the positions and definitions in his "Scattered Objects"--so it's not so surprising that I'm familiar with it!]

**Comment by JisooSeo** [comment]
Thank you, Greg, for the insightful comments!
I'd like to comment on the "boringness" that can be attributed to processes. (I'm assuming that seminars, paint drying, and others are all processes that can harbor "boringness") I'd like to highlight that boredom (borne out of a "boring" process) is subjective. Even if I accept that boringness has an appropriateness condition, I don't think it automatically renders it objective, meaning free of mind-dependency/subjectivity. Even if it is so that boringness has a cutoff at a certain point, in other words, it has some threshold of boringness, it will be wildly different from person to person where that threshold is. For instance, painting drying might be fascinating to an artist.

**Reply by gregfowlerphd** [reply]
Thanks again for replying, Jisoo!

A few things. First, I'll admit that I definitely elided over some issues in my original post. The view (which I endorse) that *being boring* is a feature whose instantiation by an entity makes it appropriate or fitting to be bored by that entity doesn't automatically entail that something can have that feature even in the absence of agents. I think you may have been pointing out this lack of entailment in your reply, in which case, point taken.

Second, I'll admit that I wasn't being entirely serious when I talked about paint drying being boring whether or not anyone's around to watch it happen. The example was a bit of a joke based on the colloquialism "like watching paint dry", and I'm not strongly committed to the view that paint drying is boring even when no one's around to be bored by it. That said, I find it pretty plausible. And maybe I'm just weird, but I also find it pretty plausible that an artist who is fascinated by paint drying is simply having an inappropriate or unfitting response.

Having said that: Third, there's nothing barring paint drying from being a component of a more complex state of affairs that isn't boring, which might help to accommodate intuitions to the effect that an artist's fascination when watching paint dry isn't inappropriate or unfitting. Perhaps the artist is fascinated not by the paint drying itself but by this more complex--and unboring!--state of affairs that has the paint drying as a component.

**Reply by JisooSeo** [reply]
That's a neat idea! 
I was also wondering if we're reacting to the same process of paint drying if we watched it at ten times the speed.

**Reply by gregfowlerphd** [reply]
Thanks! I also find your suggestion quite interesting. :)
